src/internal/connector/exchange/contact_folder_cache.go

```diff
@@ -26,13 +26,13 @@ func (cfc *contactFolderCache) populateContactRoot(
 ) error {
 	f, err := cfc.getter.GetContainerByID(ctx, cfc.userID, directoryID)
 	if err != nil {
-		return support.ConnectorStackErrorTraceWrap(err, "fetching root folder")
+		return clues.Wrap(err, "fetching root folder")
```

Comment:
I think this still needs `ConnectorStackErrorTrace` to get details
```suggestion
		return clues.Wrap(support.ConnectorStackErrorTrace(err), "fetching root folder")
```

Reply:
I haven't gotten around to updating the api package yet (being downstream, it required cascading changes to all upstream callers to implement, which is why this PR went first), but the expectation is that the API will pack that data into the error for this caller by use of [gaph.ErrData()](https://github.com/alcionai/corso/blob/main/src/internal/connector/graph/errors.go#L185).